pkg/terraform/parser.go

```diff
@@ -201,6 +201,10 @@ func (p *PlanParser) Parse(body string) ParseResult { //nolint:cyclop
 
 	changeResult := ""
 	if startChangeOutput != -1 {
+		// if we get here before finding a horizontal rule, output all remaining.
+		if endChangeOutput == -1 {
+			endChangeOutput = len(lines) - 1
+		}
```

Comment:
If [`TF_IN_AUTOMATION` environment variable](https://developer.hashicorp.com/terraform/tutorials/automation/automate-terraform#controlling-terraform-output-in-automation) is set, no horizontal rules appear after `Terraform will perform the following actions:`. 😢